hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/HoodieLogFileCommand.java

```diff
@@ -77,9 +77,14 @@ public String showLogFileCommits(
     for (String logFilePath : logFilePaths) {
       FileStatus[] fsStatus = fs.listStatus(
           new Path(logFilePath));
-      Schema writerSchema = new AvroSchemaConverter()
-          .convert(SchemaUtil
-              .readSchemaFromLogFile(HoodieCLI.tableMetadata.getFs(), new Path(logFilePath)));
+      Schema writerSchema = null;
+      try {
+        writerSchema = new AvroSchemaConverter()
+            .convert(SchemaUtil
+                .readSchemaFromLogFile(HoodieCLI.tableMetadata.getFs(), new Path(logFilePath)));
+      } catch(NullPointerException e) {
+        // unable to read schema
```

Comment:
Can we handle this more directly using null checks as needed..  the empty catch block for NPE, could be cleaner?

Reply:
Yeah, just did that to fix it quickly, cleaned it now.